src/components/recommendation/recommendation.jsx

```diff
@@ -0,0 +1,29 @@
+import PaintingTile from '../paintings-page/painting-tile/painting-tile';
+import styles from './recommendation.module.scss';
+
+export default function Recommendation({ collectionData = [] }) {
+  const paintings = collectionData.paintingsCollection
+    ? collectionData.paintingsCollection
+    : collectionData;
+
+  const classNames = {
+    recommendedContainer: styles.recommendedContainer,
+    recommendedTextContainer: styles.recommendedTextContainer,
+  };
+
+  return (
+    <div className={styles.container}>
+      <p className={styles.header}>PAINTINGS</p>
+      <p className={styles.subHeader}>You may also like</p>
+      <div className={styles.row}>
+        {paintings.map(item => {
+          return (
+            <div className={styles.painting} key={item.name}>
```

Comment:
You don't have a classname in the SCSS file called `.painting`, so this div seems unnecessary.

However, can you wrap the painting tiles in a link, so that when you click them it actually performs a redirect? You'll have to pass in some query params as well, there are examples of this in painting-info-template and collection-tile